src/Microsoft.Dnx.ApplicationHost/Program.cs

```diff
@@ -250,8 +240,6 @@ private Task<int> ExecuteMain(DefaultHost host, string applicationName, string[]
                         applicationName,
                         ex.InnerException);
                 }
-
-                throw;
```

Comment:
Isn't this `throw` still needed? I think without this we will suppress all exceptions but FileLoad/FileNotFound and compilation exceptions...